corehq/pillows/case.py

```diff
@@ -78,7 +116,7 @@ def build(self):
             elasticsearch=get_es_new(),
             index_info=CASE_INDEX_INFO,
             doc_transform=transform_case_for_elasticsearch,
-            pillow=get_case_to_elasticsearch_pillow(),
+            pillow=get_ucr_es_case_pillow(),
```

Comment:
@emord @snopoke 

Hey guys, I have a question here that I couldn't find a right solution to.

When combining multiple pillows into single pillow, the reindexer code won't work the way it is right now. The reindexers process changes using a pillow with a specific processor to the reindexer. Since now we combine the pillows, we will have to replace that with the combined pillow. There are two issues with this
1. This means that other processors also get invoked than the specific processor that the reindexer intended. 
2.  The above problem could be solved by sending a processor as a kwarg to the combined pillow, but the reindexers also set checkpoints. But since there are extra processors on the combined pillow this is not the right thing to do.

I was thinking to remove passing combined pillow to reindexers and instead let the reindexer process using just the processor. Does this sound right to you guys? Or do you have a better idea?
2.

Reply:
> To address this, I was thinking to remove passing combined pillow to reindexers and instead let the reindexer process using just the processor.

This sounds right to me. Definitely the way I would go.

Reply:
(was done a while ago)